src/Data/PQueue/Prio/Internals.hs

```diff
@@ -343,9 +325,11 @@ minViewWithKey (MinPQ n k a ts) = Just ((k, a), extractHeap n ts)
 mapWithKey :: (k -> a -> b) -> MinPQueue k a -> MinPQueue k b
 mapWithKey f = runIdentity . traverseWithKeyU (Identity .: f)
 
--- | \(O(n)\). @'mapKeysMonotonic' f q == 'mapKeys' f q@, but only works when @f@ is strictly
--- monotonic. /The precondition is not checked./ This function has better performance than
--- 'mapKeys'.
+-- | \(O(n)\). @'mapKeysMonotonic' f q == 'Data.PQueue.Prio.Min.mapKeys' f q@,
+-- but only works when @f@ is strictly monotonic.
+-- /The precondition is not checked./
+-- This function has better performance than 'Data.PQueue.Prio.Min.mapKeys'.
+
```

Comment:
```suggestion
-- | \(O(n)\). @'mapKeysMonotonic' f q == 'Data.PQueue.Prio.Min.mapKeys' f q@,
-- but only works when @f@ is strictly monotonic.
-- /The precondition is not checked./
-- This function has better performance than 'Data.PQueue.Prio.Min.mapKeys'.
```

Reply:
What am I missing?

Reply:
The change is to remove the blank line, unfortunately the diff doesn't show this.